scenes/components/health_manager.gd

```diff
@@ -0,0 +1,16 @@
+extends Node2D
+
+
+class_name HealthManager 
+
+@export var MAX_HEALTH := 100.0 
+var health: float #Fractional damage can be added 
+
+func _ready() -> void:
+	health = MAX_HEALTH
```

Comment:
This can be done in variable declaration. Saves tokens and makes the code cleaner. 
EG:
var health = MAX_HEALTH